test/integration/study_creation_test.rb

```diff
@@ -108,23 +118,48 @@ class StudyCreationTest < ActionDispatch::IntegrationTest
     assert_equal 19, study.genes.size, 'Did not parse all genes from expression matrix'
 
     # verify that counts are correct, this will ensure that everything uploaded & parsed correctly
-    study.reload
     gene_count = study.gene_count
     cluster_count = study.cluster_groups.size
     metadata_count = study.cell_metadata.size
     cluster_annot_count = study.cluster_annotation_count
     study_file_count = study.study_files.non_primary_data.size
     share_count = study.study_shares.size
 
-    assert_equal 19, gene_count, "did not find correct number of genes"
+    # I don't need this, perhaps, because this happens a little later: assert_equal 19, gene_count, "did not find correct number of genes"
```

Comment:
I would remove this if we in fact don't need it anymore.

Reply:
all set